sdk/keyvault/azure-security-keyvault-certificates/src/test/java/com/azure/security/keyvault/certificates/CertificateAsyncClientTest.java

```diff
@@ -729,11 +677,15 @@ public void getCertificateIssuer(HttpClient httpClient, CertificateServiceVersio
         createCertificateAsyncClient(httpClient, serviceVersion);
 
         getCertificateIssuerRunner((issuer) -> {
-            CertificateIssuer createdIssuer = certificateAsyncClient.createIssuer(issuer).block();
-            StepVerifier.create(certificateAsyncClient.getIssuer(issuer.getName()))
-                    .assertNext(retrievedIssuer -> {
-                        assertTrue(issuerCreatedCorrectly(createdIssuer, retrievedIssuer));
-                    });
+            AtomicReference<CertificateIssuer> certificateIssuer = new AtomicReference<>();
+            StepVerifier.create(certificateAsyncClient.createIssuer(issuer)
+                    .flatMap(createdIssuer -> {
+                        certificateIssuer.set(createdIssuer);
+                        return certificateAsyncClient.getIssuer(issuer.getName());
+                    }))
+                .assertNext(retrievedIssuer -> {
+                    assertTrue(issuerCreatedCorrectly(certificateIssuer.get(), retrievedIssuer));
```

Comment:
The recording for this test turned out empty because we are not waiting for the async operation to complete.
```suggestion
                    assertTrue(issuerCreatedCorrectly(certificateIssuer.get(), retrievedIssuer))
                    .verifyComplete();
```